### DISCUSSION TITLE
Constant DNS queries for NC domain (seeing in pihole query logs)

### DISCUSSION THREAD

**Discussion by loeffelpan** [nextcloud/all-in-one#2148] [Questions] [answered]
Something from Nextcloud AIO queries my own domain.
Looks like an healthcheck. Stopping AIO and the querys are gone.

I updated to NC26 beta due to #1846 if this is related. Couldn't say if these query were there before.

**Comment by Zoey2936** [comment]
see here: https://github.com/nextcloud/all-in-one/blob/2666313d63f4a62344903ea97f67106f9b6b8476/Containers/apache/healthcheck.sh#L8

**Reply by loeffelpan** [reply]
Shouldn't match in my reverse proxy config as `APACHE_PORT` is `11000`.

This should match:
https://github.com/nextcloud/all-in-one/blob/2666313d63f4a62344903ea97f67106f9b6b8476/Containers/apache/healthcheck.sh#L6

**Reply by Zoey2936** [reply]
But I can confirm this:
![grafik](https://user-images.githubusercontent.com/75573284/224562983-66e9028f-e808-4ed0-a8fa-23b31c116c12.png)

**Reply by loeffelpan** [reply]
OK, could not be from your `healthcheck.sh` as we investigated. But what else?
Are you on NC26? Maybe its since that from NC itself?

**Reply by loeffelpan** [reply]
I just found that collabora container does these querys. Maybe this helps.

**Reply by Zoey2936** [reply]
> OK, could not be from your `healthcheck.sh` as we investigated. But what else? Are you on NC26? Maybe its since that from NC itself?

I still use v25

**Comment by szaimen** [comment, marked as answer]
> I just found that collabora container does these querys.

Indeed

**Reply by loeffelpan** [reply]
Thanks for confirmation. But can you please explain? Never saw those queries flodding my logs before. Is that intended? For what?

**Reply by szaimen** [reply]
I think collabora requests a specific json that is used for syncing font files from your Nextcloud to your collabora.
